src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -491,4 +491,8 @@
     <value>Signed package validation failed with multiple errors:{0}</value>
     <comment>0 - new line separated list of errors</comment>
   </data>
+  <data name="Error_InvalidLockFileInput" xml:space="preserve">
+    <value>Invalid restore input where RestorePackagesWithLockFile property is set to false and also exist a packages lock file at {0}.</value>
```

Comment:
the RestorePackage... and exist**s** ?

Reply:
I copied this error message from another assembly. Should I fix it there too?

`also exist a packages lock file` sounds really weird to me. I'd like to change it to `also a packages locak file exists` in both assemblies, but not sure if we want to cause the translators extra work.